### ISSUE TITLE
Repo names nearly unreadable in dark mode

### ISSUE THREAD

**Issue by alanhogan** [ptsochantaris/trailer#335]
I think it's awesome that this app is compatible with macOS' dark mode menu bar.

The only problem is that the dark gray text for repo names (which is nice and high-contrast in light mode) is very low contrast in dark mode.

**Comment by ptsochantaris** [comment]
Hi @alanhogan 

Just to be sure I'm on the same page with what you're saying, here's a screenshot of e.g. your issue on my system (in dark mode). Are these the same colours you're seeing?

<img width="345" alt="screen shot 2017-09-21 at 08 23 09" src="https://user-images.githubusercontent.com/521581/30683761-95036fce-9ea6-11e7-98ba-1fb62bd9703c.png">

I just first want to be sure we're taking about the shade of the correct colour, and you're not seeing a bug where you're seeing a far darker version of the repo name.

If it's the case, I'll make a TODO note to make the repo names a bit lighter.

**Comment by alanhogan** [comment]
Yes, that’s exactly what mine looks like, too! thanks!

**Comment by ptsochantaris** [comment]
Great, will take care to lighten that up a bit! 👍

**Comment by ptsochantaris** [comment]
Hi @alanhogan 

Here's a build with this change in it - it will be part of the next update.

It definitely makes more sense to have the repository name brighter like you suggest since, after all, in non-dark mode the repo name _is_ darker in order to specifically stand out more.

In short, the idea was to always have the repo name be "stronger" than the other items on that line. Not sure why I originally made it darker in dark mode :)

[Trailer.zip](https://github.com/ptsochantaris/trailer/files/1338524/Trailer.zip)

Let me know how this change looks for you, and/or any further suggestions you may have.

**Comment by alanhogan** [comment]
thanks!!

Looks good to me:
<img width="398" alt="screen shot 2017-09-27 at 2 04 27 pm" src="https://user-images.githubusercontent.com/6808/30937826-d469afc6-a38c-11e7-9663-62b7cc84e225.png">

The only thing I noticed is that it’s not great contrast now for _hover_, but that doesn’t matter as much.
<img width="398" alt="screen shot 2017-09-27 at 2 04 27 pm" src="https://user-images.githubusercontent.com/6808/30937843-e431029c-a38c-11e7-9684-f59a2df4fdf6.png">

Thanks a bunch for this fix.

**Comment by alanhogan** [comment]
<img width="429" alt="screen shot 2017-09-27 at 2 04 12 pm" src="https://user-images.githubusercontent.com/6808/30937854-f321bee0-a38c-11e7-8cb5-9155da118bd0.png">

oops, used the wrong screen shot at first. updated comment

**Comment by ptsochantaris** [comment]
Ah, you're right, I missed that. In addition looking into this I was incidentally able to do some good code cleanup, finally fixed an ancient problem of scrollbar glitches, and also fixed an ugly High Sierra dark mode scrolling artefact 🎉 Seems like it's good to revisit ancient code after all :)

Here's a build with these fixes:
[Trailer.zip](https://github.com/ptsochantaris/trailer/files/1339185/Trailer.zip)

**Comment by ptsochantaris** [comment]
Hi @alanhogan 

[Version 1.6.2](https://github.com/ptsochantaris/trailer/releases/tag/1.6.2) is now up and includes these updates amongst other things. I'm going to close this issue for now but please feel free to create further issues if you spot other problems or have more suggestions!

**Comment by alanhogan** [comment]
Hey sorry, I didn’t notice the build you left here two days ago but I updated Trailer today and I noticed you enhanced the contrast even on the hover mode :) awesome. and i'm glad you were able to clean up some other things too. Really appreciate the fixes here. Super cool to see this line in the Sparkle dialog :)

<img width="620" alt="screen shot 2017-09-29 at 9 36 27 pm" src="https://user-images.githubusercontent.com/6808/31042533-6061ddfe-a55f-11e7-903e-21fe540fcbbf.png">

You rock! Have a great weekend
